pkg/dao/repository_configs.go

```diff
@@ -90,6 +90,11 @@ func (r *repositoryConfigDaoImpl) InitializePulpClient(ctx context.Context, orgI
 func (r repositoryConfigDaoImpl) Create(newRepoReq api.RepositoryRequest) (api.RepositoryResponse, error) {
 	var newRepo models.Repository
 	var newRepoConfig models.RepositoryConfiguration
+
+	if newRepoConfig.OrgID == config.RedHatOrg {
+		return api.RepositoryResponse{}, errors.New("Creating of Red Hat repositories is not permitted")
+	}
+
```

Comment:
This should be checking `newRepoReq.OrgID` instead of `newRepoConfig.OrgID` (which hasn't been set yet).

Reply:
Although I am wondering if this should instead be checked at the handler level and throw an error before getting to here

Reply:
Also you would have to do same thing for bulk create

Reply:
I'll do the same for bulk delete, I think the idea was to prevent the creation of redHat repositories at the DAO layer explicitly, as we use a separate way to create those repositories today.

Reply:
Added a check to bulk delete. 

> This should be checking `newRepoReq.OrgID` instead of `newRepoConfig.OrgID` (which hasn't been set yet).
Good catch thank you! 
I'll add a test for this.